draft-ietf-moq-transport.md

```diff
@@ -1163,6 +1191,13 @@ it can appear. If it appears in some other type of message, it MUST be ignored.
 Note that since Setup parameters use a separate namespace, it is impossible for
 these parameters to appear in Setup messages.
 
+### PUBLISHER CHOSEN SUBSCRIPTION ID {#publisher-chosen-subscription-id}
+
+The PUBLISHER CHOSEN SUBSCRIPTION ID parameter (Parameter Type 0x6) contains the
+PSID for this subscription (see {{subscription-id}}).  It can appear in
+SUBSCRIBE_OK.  If omitted, this subscription does not have a PSID and the
```

Comment:
```suggestion
SUBSCRIBE_OK, SUBGROUP, Datagram.  If omitted, this subscription does not have a PSID and the
```

Reply:
This is a Parameter to it's identifying the control messages that are valid.

Reply:
I don't think this should be in parameter.